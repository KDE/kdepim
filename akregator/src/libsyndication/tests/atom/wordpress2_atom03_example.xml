<?xml version="1.0" encoding="utf-8"?><feed version="0.3"
  xmlns="http://purl.org/atom/ns#"
  xmlns:dc="http://purl.org/dc/elements/1.1/"
  xml:lang="en"
    >
    <!-- properties:
          - Atom 0.3 by wordpress 2.0.1 
          - title, summary and content as CDATA
          -->
	<title>Foobar's Weblog</title>
	<link rel="alternate" type="text/html" href="http://www.foobar.de/wordpress" />
	<tagline>Some tagline here</tagline>
	<modified>2006-03-17T19:47:19Z</modified>
	<copyright>Copyright 2006</copyright>
	<generator url="http://wordpress.org/" version="2.0.1">WordPress</generator>
			<entry>
	  	<author>
			<name>Foobar</name>
		</author>
		<title type="text/html" mode="escaped"><![CDATA[Some title]]></title>
		<link rel="alternate" type="text/html" href="http://www.foobar.de/wordpress/archives/2006/03/17/some-title/" />
                <id>http://www.foobar.de/wordpress/archives/2006/03/17/some-title/</id>
		<modified>2006-03-17T19:44:36Z</modified>
		<issued>2006-03-17T19:44:36Z</issued>
		
	<dc:subject>asubject</dc:subject><dc:subject>anothersubject</dc:subject> 
		<summary type="text/plain" mode="escaped"><![CDATA[Some funky plain text

]]></summary>
                <content type="text/html" mode="escaped" xml:base="http://www.foobar.de/wordpress/archives/2006/03/17/some-title/"><![CDATA[<p>I like <a href="http://somewhere.com">this</a> a lot.</p>
]]></content>
<enclosure url='http://www.somewhere.com/somwhere.mp3' length='4101088' type='audio/mpeg'/>
	</entry>
	</feed>
