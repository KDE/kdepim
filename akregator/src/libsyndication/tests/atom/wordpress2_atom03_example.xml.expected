# Feed begin ######################
title: #Foobar&apos;s Weblog#
link: #http://www.foobar.de/wordpress#
description: #Some tagline here#
copyright: #Copyright 2006#
language: #en#
# Item begin ######################
id: #http://www.foobar.de/wordpress/archives/2006/03/17/some-title/#
title: #Some title#
link: #http://www.foobar.de/wordpress/archives/2006/03/17/some-title/#
description: #Some funky plain text#
content: #<p>I like <a href="http://somewhere.com">this</a> a lot.</p>#
datePublished: #Fri Mar 17 20:44:36 2006#
dateUpdated: #Fri Mar 17 20:44:36 2006#
language: #en#
# Person begin ####################
name: #Foobar#
# Person end ######################
# Item end ########################
# Feed end ########################

